### ISSUE TITLE
[BUG] SQS Service Resource Receive Messages Has Incorrect Return Type

### ISSUE THREAD

**Issue by stevenjackson121** [vemel/mypy_boto3_builder#6]
**Describe the bug**
The return type of `mypy_boto3_sqs.service_resource.Queue.receive_messages()` is actually `List[mypy_boto3_sqs.service_resource.Message]` and not `ReceiveMessageResultTypeDef`

**To Reproduce**
1. Install 'boto3-stubs[sqs]'
2.  Run mypy on the following code sample

```python
import boto3
from mypy_boto3_sqs import SQSServiceResource

service_resource: SQSServiceResource = boto3.resource("sqs")

queue = service_resource.Queue("blarghle")
message_list = queue.receive_messages()
for message in message_list:
    message.delete()
```
3. Actual mypy output

```
sqs_service_resource.py:9: error: "str" has no attribute "delete"
Found 1 error in 1 file (checked 1 source file)
```

**Expected mypy-output**

```
No Errors Found (checked 1 source file)
```

**Additional context**
Mac, `boto3-stubs` installation method, `boto3` version 1.12.12

**Comment by vemel** [comment]
Thank you. Resources are parsed completely from botocore shapes, so I did not expect any issues with them. Probably boto3 overrides some resources.

**Comment by vemel** [comment]
I found the issue. Action shape can specify a path to utput result of an operation

```python
OrderedDict([('request', OrderedDict([('operation', 'ReceiveMessage'), ('params', [OrderedDict([('target', 'QueueUrl'), ('source', 'identifier'), ('name', 'Url')])])])), ('resource', OrderedDict([('type', 'Message'), ('identifiers', [OrderedDict([('target', 'QueueUrl'), ('source', 'identifier'), ('name', 'Url')]), OrderedDict([('target', 'ReceiptHandle'), ('source', 'response'), ('path', 'Messages[].ReceiptHandle')])]), ('path', 'Messages[]')]))])
```

In this case, `path` is `Message[]`, so instead of setting the whole operation result as a return type, I have to find it inside a result.

**Comment by vemel** [comment]
Fixed in new release [1.0.4](https://github.com/vemel/mypy_boto3_builder/releases/tag/1.0.4-release) . Please update `boto3-stubs` and `boto3`.

This fixed not only `sqs`, but many other service resource methods that were returning other resources. For `sqs` there were two methods affected: `Queue.receive_messages` and `SQSServiceResource.get_queue_by_name`

Let me know if it works as it should. Thank you!

**Comment by stevenjackson121** [comment]
Everything works. Thanks for the very quick response + fix!
